warpdrive.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/sh -e
+#
+# the warp drive allows you to enter hyperspace
+
+cd "$(dirname "$0")"
+
+DOCKER="docker"
+USE_PODMAN=$(./config.py use_podman)
+
+if [ "$USE_PODMAN" == "True" ]; then
+	DOCKER="podman"
+	echo "using podman instead of docker"
+fi
+
+cat <<EOF | sudo $DOCKER exec -i docker_orbit_1 /bin/sh
```

Comment:
and then just drop sudo here. With podman, you don't nessecarily want sudo

Reply:
true